int32 sequence
Header header
Header image_header
Obstacle_Center[] obstacles_center
